### DISCUSSION TITLE
Default Keybindings improvement: Isn't one much more likely to use the Buffer symbols menu rather than the Project one?

### DISCUSSION THREAD

**Discussion by xoich** [zed-industries/zed#14142] [Feature Requests]
hello, right now the Project Symbols menu is bound to ctrl-t while the Buffer symbols to ctrl-shift-o, which is quite harder to press. Personally I use the latter much more often, at least in large projects with tons of symbols. Could swapping them be a possibility? What does anybody else think?

**Comment by Kasparsu** [comment]
At the moment searched answer to this as well 
```json
{
    "context": "Editor",
    "bindings": {
      "ctrl-t": "outline::Toggle"
    }
  }
```
fixed this

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
